.github/workflows/package.yml

```
@@ -68,14 +70,20 @@ jobs:
         id: build_and_publish
         run: |
           IMAGE_ID=${{ secrets.AWS_ACCOUNT }}.dkr.ecr.eu-west-2.amazonaws.com/dev-funding-service
```

```suggestion
          IMAGE_ID=${{ secrets.AWS_ACCOUNT }}.dkr.ecr.eu-west-2.amazonaws.com/dev-funding-service
```

The ECR name needs to be scoped to the environment, I think?